Dev multiplicities

PBrdng opened the pull request.

saschatimme commented:
@PBrdng We do no more hit the code to resolve any path crossings. Either our old test case was just bad, i.e., that we did detect a path crossing although it didn't happen or the new test doesn't detect it anymore.

PBrdng commented:
There was a bug with the input. Fixed it. Pathcrossings get detected correctly now.

I updated the multiplicities function such that a distance function is passed. We may think about whether for affine systems infinity norm for the affine vectors or Fubini Study for representatives on the sphere is better. I think the latter.
